lib/util/fonts.ts

```diff
@@ -0,0 +1,36 @@
+//font optimization from @next/font
+import { Inter } from "@next/font/google";
+import { Kanit } from "@next/font/google";
+import { Roboto_Mono } from "@next/font/google";
+
+export const inter = Inter({
+  subsets: [
+    "cyrillic",
+    "cyrillic-ext",
+    "greek",
+    "greek-ext",
+    "latin",
+    "latin-ext",
+    "vietnamese",
+  ],
+  variable: "--font-inter",
+});
+
+export const kanit = Kanit({
+  subsets: ["latin", "latin-ext", "thai", "vietnamese"],
```

Comment:
I think we just use this for the Zeitgeist logo, can we get away with just the Latin subset?

Reply:
Good call, I'll remove the other subsets.